test/test-to-s.rb

```diff
@@ -121,14 +121,14 @@ def test_20_empty_text
 
       parsed_rss = RSS::Parser.parse(rss.to_s)
       assert_equal({
-                     title: title,
-                     link: link,
-                     description: description,
+                     :title => title,
+                     :link => link,
+                     :description => description,
```

Comment:
```suggestion
                     title: title,
                     link: link,
                     description: description,
```

Not in `Style/StringLiterals`.